lib/-private/todo-handler.js

```diff
@@ -63,6 +64,22 @@ module.exports = class TodoHandler {
       }
     }
 
+    if (expiredTodos.size > 0) {
+      if (shouldFix) {
+        applyTodoChanges(getTodoStorageDirPath(this._workingDir), new Map(), expiredTodos);
+      } else {
+        for (const [, todo] of expiredTodos) {
+          results.push({
+            rule: 'expired-todo-rule',
+            message: `Expired todos exist. Please run \`ember-template-lint --clean-todo\` to remove expired todos.`,
+            filePath: todo.filePath,
+            severity: 2,
+            isFixable: true,
+          });
+        }
+      }
+    }
+
```

Comment:
I think we should evaluate this before the above conditional. Additionally, we do not want to push a new error for expired, we want to "opaquely" remove the expireds.

```suggestion
    if (expiredTodos.size > 0) {
      applyTodoChanges(getTodoStorageDirPath(this._workingDir), new Map(), expiredTodos);
    }

```